src/tests/all-tests.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+var fluid = fluid || require('infusion');
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.smtp");
+require("../js/mailserver");
+
+var jqUnit = fluid.require("jqUnit");
+var fs     = require("fs");
+
+var nodemailer    = require("nodemailer");
+var smtpTransport = require('nodemailer-smtp-transport');
+var options = {
+    secure: false,
+    port:   4026,
+    ignoreTLS: true
+};
+var transporter   = nodemailer.createTransport(smtpTransport(options));
+
+var mailOptions = {
+    from:    "sender@localhost",
+    to:      "recipient@localhost",
+    subject: "Test Subject",
+    text:    "Test Body"
+};
+
+function isSaneResponse(jqUnit, error, info) {
```

Comment:
Remember to globally namespace and appropriately name all implementation units. There is no need to pass in jqUnit as it is always unequivocally a single global.